cloud_blobstore/s3.py

```diff
@@ -149,11 +149,21 @@ def upload_file_handle(
             self,
             bucket: str,
             key: str,
-            src_file_handle: typing.BinaryIO):
+            src_file_handle: typing.BinaryIO,
+            content_type: str=None,
+            metadata: dict=None):
+        extra_args = None
```

Comment:
if `upload_fileobj()` can take `{}` as `ExtraArgs` without barfing, then this would be preferable:

```
extra_args = {}
if content_type is not None:
    extra_args['ContentType'] = content_type
if metadata is not None:
    extra_args['Metadate'] = metadata
```

Reply:
That change will work. Will fix